paddle/fluid/imperative/prepared_operator.cc

```diff
@@ -33,6 +33,8 @@ platform::Place PreparedOp::GetExpectedPlace(const platform::Place& place,
   bool found = false;
   for (auto& name_pair : ins) {
     for (auto& var_base : name_pair.second) {
+      PADDLE_ENFORCE_EQ(var_base->Var().IsInitialized(), true,
```

Comment:
move this as an else of line 39 can avoid lots of call of `PADDLE_ENFORCE`

Reply:
done